validation/telemetry/main.4.displayversion.fail.json

```diff
@@ -0,0 +1,21 @@
+{
+  "application": {
+    "architecture": "x86-64",
+    "buildId": "20151103030248",
+    "channel": "beta",
+    "name": "Firefox",
+    "platformVersion": "45.0",
+    "vendor": "Mozilla",
+    "version": "45.0",
+    "xpcomAbi": "x86_64-gcc3"
+  },
+  "creationDate": "2015-11-05T01:25:43.312Z",
+  "environment": {
```

Comment:
There is a bit more "environment" stuff that's needed to otherwise-pass, should be something like:
```
  "environment": {
    "build": {
      "applicationId": "{ec8030f7-c20a-464f-9b0e-13a3a9e97384}",
      "applicationName": "Firefox",
      "architecture": "x86-64",
      "buildId": "20151103030248",
      "displayVersion": "baddisplayversion",
      "platformVersion": "45.0",
      "vendor": "Mozilla",
      "version": "45.0",
      "xpcomAbi": "x86_64-gcc3"
    },
    "partner": {},
    "settings": {},
    "system": {}
  },
```